templates/product/index.html.twig

```diff
@@ -1,14 +1,11 @@
 {% extends 'base.html.twig' %}
 
 {% block body %}
-    <div id="app">
+    <div class="container">
+        <div class="row">
+            <div class="col-sm">
+                <h1>Hi! I would ❤️ if someone put a Vue app here...</h1>
```

Comment:
May be `Hi! I would <3 it if someone put a Vue app here...` Of course I'm not a native English speaker but to me it sounds more appropriate that way.

Reply:
Yep, your version is better!